programs/zstdcli.c

```diff
@@ -467,6 +512,8 @@ typedef enum { zom_compress, zom_decompress, zom_test, zom_bench, zom_train, zom
 
 int main(int argCount, const char* argv[])
 {
+    g_displayOut = stderr;
```

Comment:
We use declaration-after-statement, for broader compatibility with a larger class of compilers.
It forbids assigning a value to `g_displayOut` before variable declarations.

Reply:
A usual way to make an assignment before variable declaration is to create a new indentation level `{ }`.
In this case though, it would mean that the whole code inside `main()` would have to indent by 4 spaces, just to initialize `g_displayOut`, which feels overkill.

Another way could be to combine it to an assignment using the comma operator.
Typically : 
`int cLevel = ( g_displayOut = stderr, init_cLevel() );`

Only `init_cLevel()` return value will be used, but `g_displayOut` will be initialized before (the order is guaranteed).
The downside is that readability is less good, simply because less coders understand the comma operator.

Reply:
Thanks for the tip! I will just rearrange the code to make it more readable.